infrastructure/src/main/java/org/corfudb/infrastructure/InvokeCheckpointingMock.java

```diff
@@ -0,0 +1,41 @@
+package org.corfudb.infrastructure;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.DistributedCompactor;
+
+public class InvokeCheckpointingMock implements IInvokeCheckpointing {
```

Comment:
Is this a test class?  Can we move it in the test directory and add a comment if so?

Reply:
Yes it's a test class. Yes, makes sense to move it in test dir.